src/esp/bindings_js/bindings_js.cpp

```diff
@@ -51,6 +51,20 @@ std::map<std::string, ObservationSpace> Simulator_getAgentObservationSpaces(
   return spaces;
 }
 
+template <class T, typename... Targs>
+static inline auto create(Targs&&... args) {
+  return std::make_shared<T>(std::forward<Targs>(args)...);
+}
+
+Magnum::Quaternion Quaternion_fromVec4f(const vec4f& rot) {
```

Comment:
This would you could actually bind as vec4f based constructor for quaternion, no?